index.html

```diff
@@ -200,5 +200,64 @@ <h4>Skills</h4>
       </ul>
     </div>
   </section>
+  <section id="contact-form" class="flex-column">
+    <p>I'm always <br>interested in hearing about new projects, 
+      so if you'd like to chat please get in touch.</p>
+    <form action="https://formspree.io/f/xzbykgyg" method="POST">
+      <ul class="flex-column">
+        <li id="name-label">
+          <label>
+            <input type="text" id="name" name="name" placeholder="Full name" maxlength="30" 
+            required>
+          </label>
+        </li>
```

Comment:
- [ ] Kindly let's remove the list from the form, we are not obliged to use the `<ul> and <li> or <label>` to wrap the inputs inside the form,  the `<label>` can contain the name of the input but not the input itself so it should also be removed since your inputs only require the placeholder but not the label 👍